core/templates/dev/head/header_css_libs.html

```diff
@@ -26,4 +7,6 @@
 <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet">
 <link rel="stylesheet" type="text/css" media="screen"
-      href="/third_party/static/font-awesome-4.4.0/css/font-awesome.min.css">
+      href="/css/oppia.css">
```

Comment:
Actually your order works perfects. and some of css in ```oppia.css``` depended on ```third_party css```.No errors so far.